src/status_im/ethereum/subscriptions.cljs

```diff
@@ -67,16 +68,35 @@
   [{:keys [db]} _]
   {:db (assoc db :wallet-legacy/non-archival-node true)})
 
+(rf/defn wallet-owned-collectibles-filtering-done
```

Comment:
is it possible to not add code to status_im ns? - we should avoid adding new code here. What's involved with having this in status_im2 instead?